server/views/pages/reports/view/_details.njk

```diff
@@ -1,5 +1,30 @@
 {% from "govuk/components/summary-list/macro.njk" import govukSummaryList %}
 
+{% if report.descriptionAddendums.length != 0 %}
+
+  {% set descriptionHtml %}
+    <strong>{{ usersLookup[report.reportedBy].name or report.reportedBy }}</strong>
+    <br />
+    <small>{{ report.reportedAt | timeOnDate }}</small>
```

Comment:
i would recommend adding a class to make the font size stable `govuk-body-s` feels appropriate, but do check the padding and margins! otherwise maybe `govuk-!-font-size-16`

https://design-system.service.gov.uk/styles/paragraphs/#body-small
https://design-system.service.gov.uk/styles/font-override-classes/#font-size